apps/sms/views/conversation/js/conversation.js

```diff
@@ -2277,10 +2165,7 @@ var ConversationView = {
     this.enableConvertNoticeBanners();
 
     // If there was a draft, it just got sent so delete it.
-    if (this.draft) {
-      Drafts.delete(this.draft).store();
-      this.draft = null;
-    }
+    this.updateDraft();
```

Comment:
What happens here is that ```cleanFields``` above clears Composer and ```updateDraft``` removes draft since Composer and Recipients are empty, not really obvious :( Don't have better idea right now.